benchmark/buffers/buffer-base64-decode.js

```diff
@@ -12,7 +12,7 @@ function main({ n, size }) {
   // eslint-disable-next-line node-core/no-unescaped-regexp-dot
   s.match(/./);  // Flatten string.
   assert.strictEqual(s.length % 4, 0);
-  const b = Buffer.allocUnsafe(s.length / 4 * 3);
+  const b = Buffer.allocUnsafe(s.length);
   b.write(s, 0, s.length, 'base64');
```

Comment:
The former buffer size seems actually correct to me. Instead, the length should be multiplied by 3 / 4. That way the buffer is filled completely.

Reply:
I may be misunderstanding things, here, but I think the buffer is filled completely this way. Changing the allocation to `s.length - 1` results in a RangeError. So either the buffer range check is buggy, or this change is correct in the benchmark.

Reply:
No, the buffer will only be filled by three quarter. The failing range check is in line 16 (and yes, it should fail):

Let's say size is 100. The buffer allocates 300 bytes in that case. `write` now wants to write from 0 up to 400 and 400 is above the buffer length. Thus the OOR error. The actual input value is not relevant in that case.

Reply:
Ah! I see. Or at least I think so. I went with `size * 3` PTAL